_sources/GeneralIntro/ATypicalFirstProgram.rst

```diff
@@ -22,24 +22,14 @@ looks like this.
 
     print("Hello, World!")
 
-This is an example of using the **print statement**, which doesn't actually
+This is an example of using the **print function**, which doesn't actually
 print anything on paper. It displays a value on the screen. In this case, the result is the phrase:
 
 ::
 
     Hello, World!
 
-Here is the example in activecode. Remember, activecode windows allow you to execute python 
```

Comment:
I think its important that in the first couple of chapters we get students comfortable with the environment.  Is this repetitive with some other paragraphs?

Reply:
Yes, repetitive with something two pages prior, where it walks them through everything.